testcontainers/src/core/wait/mod.rs

```diff
@@ -66,6 +69,7 @@ impl WaitFor {
     }
 
     /// Wait for a certain HTTP response.
+    #[cfg(feature = "http_wait")]
```

Comment:
I think we also need to add

```suggestion
    #[cfg(feature = "http_wait")]
    #[cfg_attr(docsrs, doc(cfg(feature = "http_wait")))]

```